pkg/controller/efs/filesystem/setup.go

```diff
@@ -67,11 +74,54 @@ func SetupFileSystem(mgr ctrl.Manager, o controller.Options) error {
 		Complete(r)
 }
 
-func isUpToDate(_ context.Context, cr *svcapitypes.FileSystem, obj *svcsdk.DescribeFileSystemsOutput) (bool, string, error) {
+type custom struct {
+	kube     client.Client
+	client   svcsdkapi.EFSAPI
+	external *external
+
+	cache struct {
+		backupPolicy bool
+	}
+}
+
+func (e *custom) cacheBackupPolicyHelper(filesystemId *string) error {
+	// Get BackupPolicy
+	//default is no backup
+	policy, err := e.client.DescribeBackupPolicy(&svcsdk.DescribeBackupPolicyInput{FileSystemId: filesystemId})
+	if err != nil {
+		var awsErr awserr.Error
+		if errors.As(err, &awsErr) && awsErr.Code() == "PolicyNotFound" {
+			e.cache.backupPolicy = false
+			return nil
+		}
+		return err
```

Comment:
Should be moved into an extra function like here: https://github.com/crossplane-contrib/provider-aws/blob/master/pkg/controller/eks/addon/zz_conversions.go#L226

```suggestion
		return resource.Ignore(err, IsNotFound())
```